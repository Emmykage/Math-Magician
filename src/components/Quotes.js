import React from 'react';
import Header from './Header';

const Quotes = () => (
  <main className="container">
    <Header />
    <div className="page-layout">
      <div className="quote">
        <p>
          Mathematics is not about numbers, equation, computations and
          algorithm. its about understanding.-- Willaim paul thursdton
        </p>
      </div>

    </div>

  </main>
);
export default Quotes;
 
