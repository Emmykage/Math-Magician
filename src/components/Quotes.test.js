import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from './Quotes';

it('Quotes renders a snapshot properly', () => {
  const tree = renderer.create(<Quotes />);
  expect(tree).toMatchSnapshot();
});
