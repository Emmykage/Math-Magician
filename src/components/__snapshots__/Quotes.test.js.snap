// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes renders a snapshot properly 1`] = `
<main
  className="container"
>
  <header>
    <nav>
      <a
        className="logo"
        href="/"
        onClick={[Function]}
      >
        <h1>
          Maths Magiciacian
        </h1>
      </a>
      <ul>
        <li>
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            href="/quotes"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div
    className="page-layout"
  >
    <div
      className="quote"
    >
      <p>
        Mathematics is not about numbers, equation, computations and algorithm. its about understanding.-- Willaim paul thursdton
      </p>
    </div>
  </div>
</main>
`;
